### PR TITLE
Adds Noid minter statefile configuration.

### PR DESCRIPTION
**Pull request by rotated8**

### CODE REVIEW

**Comment by rotated8** [comment]
Yes, there should.

**Comment by rotated8** [comment]
Updated.

File: config/initializers/active_fedora-noid.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+require 'active_fedora/noid'
+
+ActiveFedora::Noid.configure do |config|
+  config.statefile = '/tmp/minter-state'
```
Comment (pmather) [inline config/initializers/active_fedora-noid.rb]: Should we change this default to `/var/sufia/minter-state` given the proposed default in InstallScripts?

**Comment by rotated8** [comment]
Updated.

**Pull request merged**